src/lib/isPrimeNumber.js

```diff
@@ -0,0 +1,10 @@
+import assertString from './util/assertString';
+
+export default function isPrimeNumber(n) {
+  assertString(n);
+  if (isNaN(n) || !isFinite(n) || n % 1 || n < 2) return false;
```

Comment:
What's the aim of `% 1`? Every finite number will return 0.

Reply:
I suppose that it tries to verify if the number is natural. For example,`2.1 % 1` will return `0.1`.

Reply:
There is a check for `isFinite` just before.

Reply:
`isFinite` returns true for decimal numbers like `2.1`

```
> isFinite(2.1)
true
> isFinite(2)
true
```

Reply:
Oh you are right, I confused finite with natural :) 
`isNaN` is still a duplicate check though. They are checked in `isFinite` if I am not wrong.

Reply:
That is only correct for the `isFinite` function defined by `underscore` and `Number`  

```
> _.isFinite(null)
false
> Number.isFinite(null)
false
> isFinite(false)
true
```